specs/example.test.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('assert');
```

Comment:
Last thing now that assert is gone (probably not worth it to set up linting on the example apps?)

```suggestion
```